src/Lumi/Styles/Box.purs

```diff
@@ -27,6 +27,11 @@ _row =
   styleModifier_
     $ css { flexDirection: str "row" }
 
+_column :: forall props. PropsModifier props
+_column =
```

Comment:
Even though `box` is a column by default, having this is useful for expressing things like
```
box
  $ _row
  $ onDesktop _column
  $ ...
```